src/DynamoCoreWpf/ViewModels/Watch3D/HelixWatch3DViewModel.cs

```diff
@@ -266,8 +266,13 @@ public LineGeometry3D Axes
 
         public PhongMaterial SelectedMaterial { get; set; }
 
-        public Transform3D Model1Transform { get; private set; }
-
+        public Transform3D BaseTransform
```

Comment:
On second thought, can we call this "SceneTransform" and add a note that says something like, "The transform applied to the entire scene."